extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -1852,53 +1856,64 @@
 
 			function walkWidgets(widgets) {
 				widgets.forEach(function(widget) {
-					if (widget.item === undefined) {
+					if (
+						widget.widgetId === undefined ||
+						smarthome.dataModel[widget.widgetId] === undefined
+					) {
 						return;
 					}
 
 					var
-						item = widget.item.name,
-						state = widget.item.state,
-						label = widget.label,
-						value = _t.extractValueFromLabel(widget.label),
-						visibility = widget.visibility,
-						labelcolor = widget.labelcolor,
-						valuecolor = widget.valuecolor;
-
-					if (value === null) {
-						value = state;
-					}
+						w = smarthome.dataModel[widget.widgetId],
+						state = "NULL",
```

Comment:
Can this ever happen?

Reply:
Normally a state is "NULL" when the item is not yet initialized.